tests/models/kosmos2/test_modeling_kosmos2.py

```diff
@@ -462,6 +464,14 @@ def check_same_values(layer_1, layer_2):
     def test_generate_from_inputs_embeds(self):
         pass
 
+    @parameterized.expand(TEST_EAGER_MATCHES_SDPA_INFERENCE_PARAMETERIZATION)
+    @require_torch_sdpa
+    @unittest.skip("KOSMOS-2 needs special preparation of inputs to get the model running")
+    def test_eager_matches_sdpa_inference(
+        self, name, torch_dtype, padding_side, use_attention_mask, output_attentions, enable_kernels
+    ):
+        pass
```

Comment:
Is there an easy way to fix? Maybe we can patch the inputs preparation function for this test run?